### PR TITLE
Fix style issues and add implicitly unwrapped optionals in remote eager support

### PR DESCRIPTION
**Pull request by bgogul**

### CODE REVIEW

**Comment by bgogul** [comment]
@swift-ci please test tensorflow

File: stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift [hunk 1]
```diff
@@ -78,11 +78,12 @@ private class TestCluster {
     // be started when we set TFE_ContextSetServerDef.
     for i in 1..<taskCount {
       debugLog("Starting task \(i)...")
-      let serverDefText = createServerDef(forTasksConfig: tasks, withTaskIndex: i)
-      let serverDef = TFE_GetServerDef(serverDefText, status)
+      let serverDefText = createServerDef(forTasksConfig: tasks, taskIndex: i)
+      let serverDef: UnsafeMutablePointer<TF_Buffer>! = TFE_GetServerDef(
+		      serverDefText, status)
```
Comment (rxwei) [inline stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift]: Spaces, not tabs.

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 2]
```diff
@@ -307,10 +307,11 @@ public final class _ExecutionContext {
 
     if case .remote(let grpcAddress) = _RuntimeConfig.session {
       debugLog("Setting up the server def to \(grpcAddress)...")
-      let serverDef = TFE_GetServerDef(grpcAddress, status)
+      let serverDef: UnsafeMutablePointer<TF_Buffer>! = TFE_GetServerDef(
+				grpcAddress, status)
```
Comment (rxwei) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: Spaces, not tabs.

File: stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift [hunk 3]
```diff
@@ -78,11 +78,12 @@ private class TestCluster {
     // be started when we set TFE_ContextSetServerDef.
     for i in 1..<taskCount {
       debugLog("Starting task \(i)...")
-      let serverDefText = createServerDef(forTasksConfig: tasks, withTaskIndex: i)
-      let serverDef = TFE_GetServerDef(serverDefText, status)
+      let serverDefText = createServerDef(forTasksConfig: tasks, taskIndex: i)
+      let serverDef: UnsafeMutablePointer<TF_Buffer>! = TFE_GetServerDef(
```
Comment (rxwei) [inline stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift]: It's a little weird to break lines this way in Swift. I'd suggest moving the whole function call to the second line.

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 4]
```diff
@@ -307,10 +307,11 @@ public final class _ExecutionContext {
 
     if case .remote(let grpcAddress) = _RuntimeConfig.session {
       debugLog("Setting up the server def to \(grpcAddress)...")
-      let serverDef = TFE_GetServerDef(grpcAddress, status)
+      let serverDef: UnsafeMutablePointer<TF_Buffer>! = TFE_GetServerDef(
```
Comment (rxwei) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: Move the whole function call to the next line.

**Review by rxwei — APPROVED** [review]

File: stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift (hunk 3 above)
Comment (rxwei) [inline stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift]: You can omit redundant type information.

```suggestion
      let serverDef: UnsafeMutablePointer! = TFE_GetServerDef(
```

File: stdlib/public/TensorFlow/CompilerRuntime.swift (hunk 4 above)
Comment (rxwei) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: Same here:

```suggestion
      let serverDef: UnsafeMutablePointer! = TFE_GetServerDef(
```

File: stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift (hunk 3 above)
Comment (bgogul) [reply stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift]: done.

Comment (bgogul) [reply stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift]: done.

File: stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift (hunk 1 above)
Comment (bgogul) [reply stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift]: done.

File: stdlib/public/TensorFlow/CompilerRuntime.swift (hunk 4 above)
Comment (bgogul) [reply stdlib/public/TensorFlow/CompilerRuntime.swift]: Done.

Comment (bgogul) [reply stdlib/public/TensorFlow/CompilerRuntime.swift]: Done.

File: stdlib/public/TensorFlow/CompilerRuntime.swift (hunk 2 above)
Comment (bgogul) [reply stdlib/public/TensorFlow/CompilerRuntime.swift]: Done.

File: stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift [hunk 5]
```diff
@@ -78,11 +78,12 @@ private class TestCluster {
...
       debugLog("Starting task \(i)...")
-      let serverDefText = createServerDef(forTasksConfig: tasks, withTaskIndex: i)
-      let serverDef = TFE_GetServerDef(serverDefText, status)
+      let serverDefText = createServerDef(forTasksConfig: tasks, taskIndex: i)
+      let serverDef: UnsafeMutablePointer! =
+          TFE_GetServerDef(serverDefText, status)
       checkOk(status)
       let server: CTFServer! = TF_NewServer(
-          serverDef!.pointee.data, serverDef!.pointee.length, status)
+          serverDef.pointee.data, serverDef.pointee.length, status)
```
Comment (rxwei) [inline stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift]:
```suggestion
        serverDef.pointee.data, serverDef.pointee.length, status)
```
Sorry for another nit: Swift's indentation is always +2, not +4.

File: stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift [hunk 6]
```diff
@@ -78,11 +78,12 @@ private class TestCluster {
     // be started when we set TFE_ContextSetServerDef.
     for i in 1..<taskCount {
       debugLog("Starting task \(i)...")
-      let serverDefText = createServerDef(forTasksConfig: tasks, withTaskIndex: i)
-      let serverDef = TFE_GetServerDef(serverDefText, status)
+      let serverDefText = createServerDef(forTasksConfig: tasks, taskIndex: i)
+      let serverDef: UnsafeMutablePointer! =
+          TFE_GetServerDef(serverDefText, status)
```
Comment (rxwei) [inline stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift]:
```suggestion
        TFE_GetServerDef(serverDefText, status)
```
Though you may be able to move it to the previous line if it's short enough.

**Review by rxwei — APPROVED** [review]

File: stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift (hunk 6 above)
Comment (bgogul) [reply stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift]: Changed the intendation. I was not able to move it to the previous line. It goes out of 80 chars.

File: stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift (hunk 5 above)
Comment (bgogul) [reply stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift]: No worries. Thanks for pointing it out. I was not sure sure about it. :)

Done.

**Pull request merged**